src/poliastro/atmosphere/models.py

```diff
@@ -0,0 +1,661 @@
+
+        Returns
+        -------
+        T: ~astropy.units.Quantity
+            Temperature at given height.
+        p: ~astropy.units.Quantity
+            Pressure at given height.
+        rho: ~astropy.units.Quantity
+            Density at given height.
+        """
+        T = self.temperature(alt, geometric=geometric)
+        p = self.pressure(alt, geometric=geometric)
+        rho = self.density(alt, geometric=geometric)
+
+        return T, p, rho
+
+
+class COESA76(COESA):
+    """ U.S Standard Atmosphere 1976.
+
+    +--------+---------+---------+-----------+---------------+---------------+
+    | Z (km) |  H (km) |  T (K)  |  p (mbar) | rho (kg / m3) | beta (K / km) |
+    +--------+---------+---------+-----------+---------------+---------------+
+    |   0.0  |   0.0   | 288.150 | 1.01325e3 |     1.2250    |      -6.5     |
+    +--------+---------+---------+-----------+---------------+---------------+
+    | 11.019 |   11.0  | 216.650 |  2.2632e2 |   3.6392e-1   |      0.0      |
+    +--------+---------+---------+-----------+---------------+---------------+
+    | 20.063 |   20.0  | 216.650 |  5.4748e1 |   8.8035e-2   |      1.0      |
+    +--------+---------+---------+-----------+---------------+---------------+
+    | 32.162 |   32.0  | 288.650 |  8.6801e0 |   1.3225e-2   |      2.8      |
+    +--------+---------+---------+-----------+---------------+---------------+
+    | 47.350 |   47.0  | 270.650 |  1.1090e0 |   1.4275e-3   |      0.0      |
+    +--------+---------+---------+-----------+---------------+---------------+
+    | 51.413 |   51.0  | 270.650 | 6.6938e-1 |   8.6160e-4   |      -2.8     |
+    +--------+---------+---------+-----------+---------------+---------------+
+    | 71.802 |   71.0  | 214.650 | 3.9564e-2 |   6.4211e-5   |      -2.0     |
+    +--------+---------+---------+-----------+---------------+---------------+
+    |  86.0  | 84.8520 |  186.87 | 3.7338e-3 |    6.958e-6   |      0.0      |
+    +--------+---------+---------+-----------+---------------+---------------+
+    |  91.0  |  89.716 |  186.87 | 1.5381e-3 |    2.860e-6   |   elliptical  |
+    +--------+---------+---------+-----------+---------------+---------------+
+    |  110.0 | 108.129 |  240.00 | 7.1042e-5 |    9.708e-8   |      12.0     |
+    +--------+---------+---------+-----------+---------------+---------------+
+    |  120.0 | 117.777 |  360.00 | 2.5382e-5 |    2.222e-8   |  exponential  |
+    +--------+---------+---------+-----------+---------------+---------------+
+    |  500.0 | 463.540 |  999.24 | 3.0236e-9 |   5.215e-13   |  exponential  |
+    +--------+---------+---------+-----------+---------------+---------------+
+    | 1000.0 | 864.071 |   1000  | 7.5138e-5 |   3.561e-15   |  exponential  |
+    +--------+---------+---------+-----------+---------------+---------------+
+
+    """
+
+    def __init__(self):
+
+        # Lower layers (Z < 86km)
+        Hb_lower = [0.0, 11.0, 20.0, 32.0, 47.0, 51.0, 71.0] * u.km
```

Comment:
Any chance these valuable constants can be stored outside of the class?

Reply:
No problem! For the moment I will just move them outside the class but in the future, it may be interesting to have them stored in some kind of `*.dat` files as discussed privately